.github/validation_samples/reduced_ldmx/config.py

```diff
@@ -16,7 +16,7 @@
 myGun.enablePoisson = False #True   
 
 mySim = sim.simulator( "mySim" ) # Build simulator object
-det = 'ldmx-reduced-v2'
+det = 'ldmx-reduced-v1'
```

Comment:
```suggestion
det = 'ldmx-reduced-v2'
```
I think we should keep this on v2